src/pages/actions/events.ts

```diff
@@ -33,6 +34,10 @@ export type ReducerEvent =
         error: any;
       };
     }
+  | {
+      type: "ledgerActionSuccessful";
```

Comment:
:wrench: The button is called `confirm`. This one could use the same language and say `ledgerActionConfirmed`.